### ISSUE TITLE
additional documentation

### ISSUE THREAD

**Issue by nvdaTrac** [nvaccess/test_nvdaTracImport#254]
**Reported by jrutkowski on 12 Sep 2008 13:40 UTC**
When entering a program like Skype for example, NVDA doesn't have a keystroke that brings up available hotkeys within the current program.
This would be very benneficial to newbies who are just getting started with this screen reader.

**Comment by nvdaTrac** [comment]
**Comment by Iris on 12 Sep 2008 23:44 UTC**
However, as far as I know, Skype itself doesn't have specialized hotkeys for it. All the standard functions use standard controls, i.e. answering calls use a submenu etc.

**Comment by nvdaTrac** [comment]
**Comment by jteh on 18 Sep 2008 20:54 UTC**
I don't believe NVDA should provide information on keyboard shortcuts within the application. This should be provided by the application documentation and users should become familiar with reading this. I suspect this was the intent of the ticket, so closing as wontfix.

NVDA should only provide information on NVDA specific keystrokes. We don't have many (any?) of these for specific applications, so right now, the keyboard quick reference should be enough. We do need to implement the ability to dynamically provide a summary of NVDA shortcut keys, though.
